app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,7 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    highest_supported_api_version && highest_supported_api_version >= '4'
```

Comment:
I find `>=` on Strings dangerous and strange.
Also `highest_supported_api_version` seems to return `3` (int) as a default.... 
Consider some of those conversions:

```
[3] pry(main)> 3 >= '4'
ArgumentError: comparison of Fixnum with String failed
from (pry):3:in `>='
[4] pry(main)> 3.to_i >= 4
=> false
[5] pry(main)> '3.0'.to_i
=> 3
[6] pry(main)> Integer('3.0')
ArgumentError: invalid value for Integer(): "3.0"
from (pry):6:in `Integer'
[7] pry(main)> '3.0beta'.to_i
=> 3
```